src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,12 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return result1 == bet1 && result2 == bet2 ? 2 :
```

Comment:
try to simplify this solution :)

Reply:
Should I try to simplify all the solution or only 18 line?